python-package/mc2client/core.py

```diff
@@ -603,7 +604,7 @@ def generate_symmetric_key(num_bytes=32):
         )
         return
 
-    key = AESGCM.generate_key(bit_length=num_bytes * 8)
+    key = AESGCM.generate_key(bit_length= _LIB.cipher_key_size() * 8)
```

Comment:
Can you please create a variable `num_bytes` and set it to `_LIB.cipher_key_size()` for clarity? This will add one more line but should make things easier to read.

Reply:
of course I'm doing it right now